drivers/bridge/setup_ip_tables.go

```diff
@@ -178,7 +178,11 @@ func (n *bridgeNetwork) setupIPTables(ipVersion iptables.IPVersion, maskedAddr *
 			return iptable.ProgramChain(filterChain, config.BridgeName, hairpinMode, false)
 		})
 
-		n.portMapper.SetIptablesChain(natChain, n.getNetworkBridgeName())
+		if ipVersion == iptables.IPv4 {
+			n.portMapper.SetIptablesChain(natChain, n.getNetworkBridgeName())
```

Comment:
this snippet LGTM, can you please raise a separate PR with this diff, so we can get it merged sooner

Reply:
I can do but without the changes for the proxy the feature will still not be usable so I am not sure if it is useful.

Reply:
`userland-proxy` can be disabled from the daemon side https://docs.docker.com/engine/reference/commandline/dockerd/#daemon
so it should help out a bit

Reply:
I created #2603 with the first 2 commits of this branch

Reply:
I see it now :(, its not a regression but setting `--ip6tables` breaks port forwarding